python_modules/libraries/dagster-aws/dagster_aws/utils/mrjob/retry.py

```diff
@@ -101,7 +101,7 @@ def __getattr__(self, name):
 
         if name in self.__unwrap_methods:
             return partial(x.__func__, self)
-        elif hasattr(x, "__call__"):
```

Comment:
this is vendored code so i would hesitate to make random changes to it

Reply:
@mgasner removed the fix!